math/mobius/mobius_test.go

```diff
@@ -0,0 +1,49 @@
+// mobius_test.go
+// description: Returns μ(n)
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see mobius.go
+
+package mobius
+
+import (
+	"testing"
+)
+
+func getTestsForMu() []struct {
+	n        int
+	expected int
+} {
```

Comment:
I would much rather prefer you define a new struct instead of an anonymous struct on the return type...

```suggestion
type testRes struct {
	n        int
	expected int
}

func getTestsForMu() []testRes {
```
or something along those lines...

Reply:
Fixed [here](https://github.com/TheAlgorithms/Go/pull/532/commits/af9e45b37abe3ec7372f60bec064f063e932e18a)